src/events/sql.js

```diff
@@ -1,5 +1,5 @@
 const uuid4 = require('uuid4');
-const { parse } = require('../resource_utils/sql_utils.js');
+const { parse } = require('../resource_utils/sqs_utils');
```

Comment:
Why?

Reply:
had errors on that line trying to run the tests, said it couldn't find the
module that change fixed it